spec.md

```diff
@@ -586,18 +589,66 @@ schema being used.
           "40 concurrent connections"
         ]
       }
+    }, {
+      "name": "fake-deprecated-plan",
+      "id": "009aba-XXXX-XXXX-XXXX-dace631cd648",
+      "description": "This is deprecated.",
+      "free": false,
+      "deprecation": {
+        "description": "Don't use this anymore.",
+        "since": "2018-04-20",
+        "eol": "2020-02-28",
+        "alternatives": [{
+            "description": "Use this plan instead.",
+            "plan_id": "0f4008b5-XXXX-XXXX-XXXX-dace631cd648"
+          }
+        ]
+      }
     }]
   }]
 }
 ```
 
+##### Deprecation Object 
+
+| Response Field | Type | Description |
+| --- | --- | --- |
+| description | string | A short description of the deprecation. |
+| since | date string | ISO 8601 formatted date when the Service Broker deprecated the service or plan. |
+| eol | date string | ISO 8601 formatted date when the Service Broker expects to remove the service or plan from the catalog. |
```

Comment:
either say what "eol" means (end of life), or say it in words "removeDate". I kind of like being explicit, so we should change "since"  to "deprecratedDate".

Reply:
:( I kinda like `eol` and `since`, they have context because of the object they are apart of. I thought these were pretty standard usage from my past. defining eol is fine.